content/en/logs/troubleshooting/live_tail.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Live Tail
+description: 'Troubleshoot Logs Live Tail errors or issues'
+---
+
+If your Live Tail page shows an error, or the logs do not load, try the following troubleshooting steps:
+
+* Open the Live tail view with your browser in Incognito mode, and see if you can view logs.
+* Try a different browser, and check if Live Tail loads logs.
+* Check whether other team members can see logs in the Live Tail view.
+* Check for any network restrictions, VPN configurations, or antivirus software that may block the Live Tail from loading logs:
+	- Ensure that your local or company network allows inbound traffic from `live.logs.datadoghq.com`.
+	- Find the Datadog IP prefixes at the [IP ranges][1] endpoint.
+
+## Trace requests
+
+Traceroute is a tool that helps you test the path a packet takes from source to destination. You can use `traceroute` to identify any client-side network issues that may stop Live Tail from loading.
+
+To examine the route for live tail logs on Linux or MacOS, run the following command:
+{{< code-block lang="shell">}}
+traceroute live.logs.datadoghq.com
+{{< /code-block >}}
+
+If at any point the request times out, that means the request is blocked somewhere between the client and Datadog servers. Check with your network administration team to address this issue.    
+
+If a step in the route shows `* * *`, then a host did not reply to the traceroute, or a router does not reply to that protocol. The `* * *` pattern does not always indicate a timeout. Changing the discovery protocol to ICMP/PING using the `-I` option may provide more detailed results.
```

Comment:
```suggestion
If a step in the route shows `* * *`, this could indicate a host did not reply to the traceroute, or a router does not reply to that protocol. The `* * *` pattern does not always indicate a timeout. Changing the discovery protocol to ICMP/PING using the `-I` option may provide more detailed results.
```
Just a suggestion for flow, but your call!